production/frontend/src/tests/components/production_pages/monitor_pages/monitor_home_page.test.js

```diff
@@ -112,4 +139,109 @@ describe("Separators test suite", () => {
     expect(res.releasedDelayed30).toEqual(0);
     expect(res.releasedDelayed30Plus).toEqual(1);
   });
+
 });
+
+describe("ActivityOrders Separators test suite", () => {
+
+  const state = new TracershopState();
+  Object.assign(state, testState);
```

Comment:
These lines are technically not needed due to the TracershopState being const by convention, so you can just use testState.

But this is the niddiest of nid-pick.

Reply:
Oh, I just matched how I saw testState being used in other test files.